File: hello.java
import java.util.Scanner;

public class hello {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        // lê duas variáveis inteiras
        int a = in.nextInt();
        int b = in.nextInt();

        // escreve o resultado
	if (a > b)
	    System.out.printf("%d\n", a);
	else
	    System.out.printf("%d\n", b);
    }
}

File: hello.cpp
#include<iostream>
using namespace std;

int val1, val2;

int main(void) {
  // lê as duas vals
  cin >> val1 >> val2;

  // escreve o resultado
  if (val1 > val2)
    cout << val1 << endl;
  else
    cout << val2 << endl;

  // termina a execução
  return 0;
}